src/librustdoc/html/markdown.rs

```diff
@@ -776,10 +775,10 @@ impl LangString {
                     data.no_run = true;
                 }
                 x if x.starts_with("edition") => {
-                    data.edition = x[7..].parse::<Edition>().ok();
+                    data.edition = x.strip_prefix("edition").unwrap().parse::<Edition>().ok();
```

Comment:
re https://github.com/rust-lang/rust/pull/74048/files#r449881344

I changed the code to use `strip_prefix` to avoid panicking code caused by slicing.